pkg/controllers/restore/controller.go

```diff
@@ -472,30 +503,51 @@ func (h *handler) generateDependencyGraph(ownerToDependentsList map[string][]res
 }
 
 func (h *handler) createFromDependencyGraph(ownerToDependentsList map[string][]restoreObj, created map[string]bool,
-	numOwnerReferences map[string]int, objFromBackupCR ObjectsFromBackupCR, toRestore []restoreObj) error {
+	numOwnerReferences map[string]int, objFromBackupCR ObjectsFromBackupCR, toRestore []restoreObj, restoreWebhooks bool) ([]restoreObj, error) {
 	numTotalDependents := 0
 	for _, dependents := range ownerToDependentsList {
 		numTotalDependents += len(dependents)
 	}
+
+	mutatingWebhookGVR := schema.GroupVersionResource{
+		Group:    arv1.SchemeGroupVersion.Group,
+		Version:  arv1.SchemeGroupVersion.Version,
+		Resource: mutatingWebhookConfigurationResource,
+	}
+	validatingWebhookGVR := schema.GroupVersionResource{
+		Group:    arv1.SchemeGroupVersion.Group,
+		Version:  arv1.SchemeGroupVersion.Version,
+		Resource: validatingWebhookConfigurationResource,
+	}
+
 	countRestored := 0
 	var errList []error
+	var webhookList []restoreObj
 	for len(toRestore) > 0 {
 		curr := toRestore[0]
 		if len(toRestore) == 1 {
 			toRestore = []restoreObj{}
 		} else {
 			toRestore = toRestore[1:]
 		}
-		if created[curr.ResourceConfigPath] {
-			logrus.Infof("Resource %v is already created/updated", curr.ResourceConfigPath)
-			continue
-		}
 		currResourceInfo := objInfo{
 			Name:       curr.Name,
 			Namespace:  curr.Namespace,
 			GVR:        curr.GVR,
 			ConfigPath: curr.ResourceConfigPath,
 		}
+
+		// Check if the resource is a webhook, and check if the resource has been created/updated already.
+		if !restoreWebhooks && (currResourceInfo.GVR == mutatingWebhookGVR || currResourceInfo.GVR == validatingWebhookGVR) {
```

Comment:
```suggestion
		if restoreWebhooks && (currResourceInfo.GVR == mutatingWebhookGVR || currResourceInfo.GVR == validatingWebhookGVR) {
```

Reply:
I believe this is correct actually. We want to skip if we are..

1. _not_ restoring webhooks
2. the current resource is a webhook

If those conditions are met, we skip restoring the resource and append it to the list of webhooks.

Reply:
The above explanation is much clear! 
Can you update the comment in the code with your above explanation?